deployment/live/aws/codelab/README.md

```diff
@@ -91,18 +119,48 @@ with a running EC2 Amazon Linux VM, and the following software installed:
               --out_priv=/home/ec2-user/tessera-keys/$LOG_NAME.sec \
               --out_pub=/home/ec2-user/tessera-keys/$LOG_NAME.pub
     ```
- 1. Run the conformance binary in `trillian-tessera/cmd/conformance/aws`.
-    This binary is a small personality that accepts `add/` requests,
-    and stores the data in the Trillian Tessera infrastructure you've
-    just brought up:
+
+ 1. Generate and copy these environment variale definitions. \
+    They will allow you
+ to send requests to the log from a different terminal once it's running:
+    ```
+    echo -e "\n\n"
+    echo =================================================================================================================
+    echo "Copy these variable definitions to use in a different terminal:"
+    echo -e "\n"
+    echo "export WRITE_URL=http://localhost:2024/"
+    echo "export READ_URL=https://$LOG_BUCKET.s3.$AWS_REGION.amazonaws.com/"
+    echo "export LOG_PUBLIC_KEY=$(cat /home/ec2-user/tessera-keys/$LOG_NAME.pub)"
+    echo =================================================================================================================
+    echo -e "\n\n"
+    ```
```

Comment:
I found this a bit confusing, but I understand what you're doing now. How about tweaking the wording a bit & removing unnecessary fluff in the `bash`?

Also, consider moving this section below the "Run the conformance binary" step if possible - I think that'll help with mental "flow" through these steps.

> Running the commands below will print some easily copy-and-pasteable `export`s which you can use to set up the environment in a second terminal ready to be able to send requests:

```suggestion
    echo "export WRITE_URL=http://localhost:2024/"
    echo "export READ_URL=https://$LOG_BUCKET.s3.$AWS_REGION.amazonaws.com/"
    echo "export LOG_PUBLIC_KEY=$(cat /home/ec2-user/tessera-keys/$LOG_NAME.pub)"
```

Reply:
I like your wording, thanks. It can't be moved below because Tessera will be running, so folks would have to kill Tessera in order to run these commands. Alternatievly, we could kick Tessera in the background but, that feels worse.